pds_doi_service/core/input/input_util.py

```diff
@@ -192,33 +192,10 @@ def _parse_rows_to_doi_meta(self, xl_sheet):
 
         for index, row in xl_sheet.iterrows():
             logger.debug(f"row {row}")
```

Comment:
Unrelated to the current PR @collinss-jpl @jordanpadams but I'm mentioning it anyway:

Use `logger.debug('row %s', row)` (or `%d` or `%r`) instead of an `f` string here. You want the logger to do the formatting.

Why? If the `DEBUG` level of logging is *off*, there's no need to format the string, which the logger will skip doing if it's not necessary. Using an `f` string or string `.format` or string interpolation makes that awlays happen outside of the logger, meaning unnecessary cycles.

Reply:
@nutjob4life good to know, thanks for the feedback (I am using the f string everywhere !)